lib/xapi/problem.rb

```diff
@@ -63,7 +67,12 @@ def md
       path('%s.md' % slug)
     end
 
-    def path(f)
+    def path(file_name)
+      old_path = root_path(file_name)
+      File.exist?(old_path) ? old_path : File.join(meta_dir, file_name)
+    end
```

Comment:
How about "deprecated_path" instead of "old_path"? That will also clarify the ordering for the reader: we want to find it at the path, but if it's not there, then we'll use the deprecated path.